scripts/runtime/Freeze-RuntimeInputPacket.ps1

```
@@ -148,31 +158,16 @@ function Test-VibeSingleOptionCanonicalConfirmSurface {
     return $true
 }
 
-function Test-VibeProgressiveEntryLegacyConfirmBypass {
+function Resolve-VibeControllerEntryIntentId {
     param(
-        [AllowNull()] [object]$RouteResult,
         [AllowEmptyString()] [string]$EntryIntentId = '',
-        [AllowEmptyString()] [string]$TaskType = ''
+        [AllowEmptyString()] [string]$RuntimeSelectedSkill = ''
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'RuntimeSelectedSkill' has been declared but not used.

(PSReviewUnusedParameter)